pdns/recursordist/rec_channel_rec.cc

```diff
@@ -1850,12 +1857,14 @@ static RecursorControlChannel::Answer help()
           "add-nta DOMAIN [REASON]          add a Negative Trust Anchor for DOMAIN with the comment REASON\n"
           "add-ta DOMAIN DSRECORD           add a Trust Anchor for DOMAIN with data DSRECORD\n"
           "current-queries                  show currently active queries\n"
+          // "clear-cookies                    clear cookie table\n" XXX undocumented for now
           "clear-dont-throttle-names [N...] remove names that are not allowed to be throttled. If N is '*', remove all\n"
           "clear-dont-throttle-netmasks [N...]\n"
           "                                 remove netmasks that are not allowed to be throttled. If N is '*', remove all\n"
           "clear-nta [DOMAIN]...            Clear the Negative Trust Anchor for DOMAINs, if no DOMAIN is specified, remove all\n"
           "clear-ta [DOMAIN]...             Clear the Trust Anchor for DOMAINs\n"
           "dump-cache <filename> [type...]  dump cache contents to the named file, type is r, n, p or a\n"
+          "dump-cookies <filename>          dump the contents of the cookie data to the namewd file\n"
```

Comment:
```suggestion
          "dump-cookies <filename>          dump the contents of the cookie store to the namewd file\n"
```

also, shouldn't we call it a jar?

Reply:
I'll ponder that. I also spotted a typo.